src/qweb/compiler.ts

```diff
@@ -28,8 +29,8 @@ export type TemplateFunction = (blocks: any, utils: any) => Template;
 
 type BlockType = "block" | "text" | "multi" | "list" | "html";
 
-export function compileTemplate(template: string, name?: string): TemplateFunction {
-  const compiler = new QWebCompiler(template, name);
+export function compileTemplate(template: string, name?: string, translateFn?: (s: string) => string): TemplateFunction {
```

Comment:
don't like that API

Reply:
probably better to put the name and the translateFn as option